app/src/main/java/org/gdg/frisbee/android/common/GdgNavDrawerActivity.java

```diff
@@ -401,7 +400,7 @@ private void maybeUpdateChapterImage() {
                     @Override
                     public Person doInBackground(String... params) {
 
-                        return PlusPersonDownloader.getPersonSync(params[0]);
+                        return App.getInstance().getPlusApi().getPerson(params[0]);
```

Comment:
Since it was difficult to work with Plus Client, it was not easy to have sync and async calls at the same time. 

Now that we have retrofit, we can just use Call class. If we use that we can call the network async or sync. If we can get rid of this AsyncTask and have async retrofit call here, that would be great. That can also be a TODO if you don't want to do it. 

What do you think?